### PR TITLE
Review submission guidelines and double-blind instructions (issues #4 and #5)

### PR DESCRIPTION
**Pull request by fermadeiral**

### CODE REVIEW

File: cfp.html [hunk 1]
```diff
@@ -160,8 +160,8 @@ <h5>Evaluation Criteria</h5>
                             <br>
                             <h5>Double-Blind Review Process</h5>
                             <p align="justify">
-                                VEM 2021 will employ a double-blind review process.
-                                Authors should check the submission guidelines below.
+                                VEM 2021 will employ a double-blind review process. Both the authors and the reviewers must make every effort to honor the double-blind review process. Authors should check the submission guidelines below. Reviewers should not search for the title of
+                                the submitted papers on the Internet or try to deduce the authors of the papers.
```
Comment (felipeebert) [inline cfp.html:164]: "nor" try to deduce

File: cfp.html [hunk 2]
```diff
@@ -170,27 +170,28 @@ <h5>Double-Blind Review Process</h5>
                         <h2>Submission Guidelines</h2>
                         <div class="description-one">
                             <p align="justify">
-                                The VEM workshop accepts technical papers (up to TODO pages) written in English or in Portuguese. All papers must be submitted in PDF format and formatted according to the SBC template (available at TODO).
+                                <b>The format for submissions has changed for VEM 2021. Therefore, read carefully the instructions below.</b>
```
Comment (felipeebert) [inline cfp.html:173]: "please" read

File: cfp.html [hunk 3]
```diff
@@ -170,27 +170,28 @@ <h5>Double-Blind Review Process</h5>
...
                             </p>
                             <br>
                             <p align="justify">
-                                Submitted papers must be unpublished and should not be under review elsewhere. Experimental evaluation of the results is desirable, but it is not a precondition to submit a paper to the workshop. Papers with innovative and promising proposals are welcome.
-                                The acceptance of a paper implies that at least one of its authors will register for the workshop and present the paper. The workshop proceedings will be published online available from CBSoft 2021 page. Papers should be
-                                submitted electronically through TODO.
+                                Papers can be written in Portuguese or English. Submission in English is strongly encouraged so that your research can be accessed by non-Portuguese speaking researchers. Submissions must be no longer than 5 pages including all text, tables, figures,
```
Comment (felipeebert) [inline cfp.html:177]: "Submissions" in English

File: cfp.html [hunk 4]
```diff
@@ -170,27 +170,28 @@ <h5>Double-Blind Review Process</h5>
...
+                                LaTeX users must use the <code>sample-vem.tex</code> file as the <code>main.tex</code> file. Word users should use <code>interim-layout-vem.docx</code>.
                             </p>
                             <br>
                             <p align="justify">
-                                For the double-blind review process, both author names and affiliations must be omitted from the manuscript. Besides, there are other rules that have to be considered to ensure the manuscript is correctly prepared for double-blind
-                                review:
+                                For the double-blind review process, submissions must not reveal the authors' identities. In particular, the authors’ names and affiliations must be omitted in the title page and hearders of the paper, and references to their prior work should be in the
+                                third person. If you want to make available any artifact in time for submission, anonymize it. There are existing tools for doing that easily, for instance, <a href="https://anonymous.4open.science/">Anonymous GitHub</a>,
```
Comment (felipeebert) [inline cfp.html:184]: "please make it anonymous".

File: cfp.html [hunk 5]
```diff
@@ -170,27 +170,28 @@ <h5>Double-Blind Review Process</h5>
...
-                                <li>You should not relying on supplementary material. However, whether you need to make available any artifact, please anonymize it and the respective website.</li>
-                            </ul>
+                            <br>
                             <p align="justify">
-                                In case you have any doubt about the preparation of your manuscript following the double-blind rules, do not hesitate to send an email to <a href="mailto:TODO">TODO</a>.
+                                Papers must be submitted electronically through EasyChair at <a href="https://easychair.org/conferences/?conf=vem2021">this page</a>. Submitted papers must be unpublished and should not be under review elsewhere.
+                            </p>
+                            <br>
+                            <p align="justify">
+                                In case you have any doubt about the preparation of your manuscript, do not hesitate to send an email to <a href="mailto:vemworkshop@gmail.com">vemworkshop@gmail.com</a>.
```
Comment (felipeebert) [inline cfp.html:194]: any "doubts"... "please" do not hesitate

**Review by felipeebert — COMMENTED** [review]
I added some minor comments

**Comment by fermadeiral** [comment]
@felipeebert I did a commit with the requested changes. Please, let me know if there is something else that should be changed.

**Pull request merged**